### DISCUSSION TITLE
Breaking changes Home Assistant and MQT

### DISCUSSION THREAD

**Discussion by Bishbosh02** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#292] [General]
FYI
https://www.home-assistant.io/blog/2022/06/01/release-20226/#breaking-changes

**Comment by visualapproach** [comment]
Got it 👍

**Comment by chunkysteveo** [comment]
I heard on the HASS podcast that auto discovery will work fine still, which is good to know!
